### PR TITLE
Add ROI import to ImageJ overlay

### PR DESCRIPTION
**Pull request by stelfrich**
The PR adds a choice for the user to determine whether to import ROIs to the ImageJ ROI manager or create a new overlay and add them to that. The choice is disabled if the _Display ROIs_ is disabled.

### CODE REVIEW

**Comment by stelfrich** [comment]
There are failing tests for the merge to develop: https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-full-repository/130/changes

I am a little lost on how to investigate these issues..

**Comment by sbesson** [comment]
@stelfrich: the failures in the job are due to `OutOfMemory` exceptions (see https://github.com/openmicroscopy/bioformats/pull/2282). More generally, the repository jobs are testing regressions due to changes in the readers under `formats-gpl/formats-bsd`. As your changes are primarily modifying the ImageJ/Fiji plugin, you can safely ignore these failures in the context of this PR. Thanks for the active checking though.

**Comment by dgault** [comment]
@stelfrich: Hi Stefan, I have finished testing out your PR today. The feature certainly looks like it could be very useful and the dialog additions and overlay option look good and are all working well.

The only issue I encountered was when using the ROI Manager selection, the ROI Handler wasnt  recognising the selection in roisMode.equals(ImporterOptions.ROIS_MODE_MANAGER) 

This can be resolved if you update the 2 instances of the above in ROI Handler with:
roisMode.equalsIgnoreCase(ImporterOptions.ROIS_MODE_MANAGER)

**Comment by stelfrich** [comment]
Thanks for the feedback, @dgault! I have integrated your comment.

> The only issue I encountered was when using the ROI Manager selection, the ROI Handler wasnt recognising the selection in `roisMode.equals(ImporterOptions.ROIS_MODE_MANAGER)`

How did you notice (just out of curiosity)? Everything works fine for me with `equals`.

**Comment by dgault** [comment]
Not sure what the difference was but for me the default value for the option was "ROI Manager" and the ROIS_MODE_MANAGER Sting was using "ROI manager" with a lowercase m. 

I've retested with the update and everything is working as expected. PR looks good to merge from my perspective.

File: components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt [hunk 1]
```diff
@@ -189,9 +189,19 @@ default = false
...
-  Adds any ROIs in the file to ImageJ's ROI manager.
+  Adds any ROIs in the file to ImageJ.
 default = false
 
+[roiMode]
+label = ROIs_Import Mode:
+info = <b>ROI Import Mode</b> - \
+  ROIs can be imported to the ROI manager or an overlay.
+type = string
+default = ROI Manager
```
Comment (sbesson) [inline components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt]: On top of the `equalsIgnoreCase` fix added in https://github.com/openmicroscopy/bioformats/pull/2333/commits/0e45bd4b44e61dd57404a35418b496a157159e88, is it worth using the same nomenclature as `ROIS_MODE_MANAGER` for `default` and `values` i.e. `ROI manager` with lower case `m`?

Comment (stelfrich) [inline components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt]: I also found that yesterday, @sbesson, but decided to leave the fix in place. Would you rather have the naming consistent with `equals`?

Comment (sbesson) [inline components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt]: @stelfrich: thinking mostly in terms of maintenance, imposing a consistent naming scheme might minimize similar issues if the new `roiMode` gets reused anywhere else so I would vote for that.

Comment (stelfrich) [inline components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt]: I have updated the PR to fix that inconsistency. Thanks for the feedback, @sbesson!

**Comment by sbesson** [comment]
Retested with the last merge build with a fake file with ellipses

screen shot 2016-04-14 at 08 53 56

Everything works as expected. Thanks @stelfrich.

**Pull request merged**